types/dockerode/index.d.ts

```diff
@@ -894,8 +918,37 @@ declare namespace Dockerode {
   }
 
   interface ImageBuildContext {
-  	context: string;
-  	src: string[];
+    context: string;
+    src: string[];
+  }
+
+  interface DockerVersion {
+    ApiVersion: string;
+    Arch: string;
+    BuildTime: Date;
+    Components: Array<{
+      Details: {
+        ApiVersion: string;
+        Arch: string;
+        BuilTime: Date;
```

Comment:
Typo here
```suggestion
        BuildTime: Date;
```

Reply:
Nice catch, however I just noticed that something got screwed up during the pull request. Note that the DockerVersion interface exists 2 times, because it was added one time with the pull request.
This was never intended.

Reply:
Ah I didn't notice the second instance of this, I thought it didn't exist and you added it in as part of these changes.

Reply:
I removed the second instance. Also, just noticed, that the typo does currently exist in the master branch: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/dockerode/index.d.ts#L909